examples/seismic/viscoacoustic/operators.py

```diff
@@ -367,13 +363,14 @@ def deng_2nd_order(model, geometry, p, **kwargs):
     # Auxiliary Function
     l = VectorFunction(name="l", grid=model.grid, space_order=space_order)
 
+    h = TimeFunction(name="h", grid=model.grid, time_order=2, space_order=space_order,
```

Comment:
@FabioLuporini  these auxiliary functions are temporaries.
Some of them are like `TimeFunction` / `VectorTimeFunction`, however it would not be necessary. Just `Function` / `VectorFunction` should be enough. But the last one generates a segmentation fault.